ansible_collections/arista/avd/examples/cv-pathfinder/group_vars/TRANSPORTS.yml

```diff
@@ -0,0 +1,62 @@
+---
+type: spine
+
+bgp_as: 65000
```

Comment:
For the sake of it, I would like the two clouds to use different ASNs. And not the same as our WAN.